RoFILib/configuration/Configuration.h

```diff
@@ -36,8 +36,24 @@ class Module {
     ID getId() const { return id; }
     double getJoint(Joint a) const;
 
-    // return value specifies if the module was changed
+    /**
+     * \brief Rotates joint
+     * 
+     * \return `false` if adding \p val makes value of \p joint 
+     * overflow out of accepted range and value of joint isn't changed
```

Comment:
Better to put it in a separate sentence in the middle of body. "Failure in this method leaves the joint unchanged"